blog/views.py

```diff
@@ -20,4 +21,14 @@ def post_edit(request, pk):
         return JsonResponse(data={}, status=HTTPStatus.BAD_REQUEST)
     else:
         post.save()
-    return JsonResponse(data={}, status=HTTPStatus.OK)
\ No newline at end of file
+    return JsonResponse(data={}, status=HTTPStatus.OK)
+
+
+@require_http_methods(["DELETE"])
+def post_delete(request, pk):
+    try:
+        post = Post.objects.get(pk=pk)
+    except Post.DoesNotExist:
+        return JsonResponse(data={}, status=HTTPStatus.NOT_FOUND)
+    post.delete()
```

Comment:
IMHO,

이렇게 써도 코드 이해하는데 크게 무리가 없지만, 이 경우에는 else 내에 post.delete() 를 써줘서 `post에 대한 Error 가 발생하지 않았을 때, 삭제를 한다` 라는 의도를 드러내 주는게 좋을 것 같습니다~

```suggestion
    try:
        post = Post.objects.get(pk=pk)
    except Post.DoesNotExist:
        return JsonResponse(data={}, status=HTTPStatus.NOT_FOUND)
    else:
        post.delete()
```

Reply:
7eac3f7 - 반영했습니다